static/docs/commands-reference/destroy.md

```diff
@@ -5,7 +5,9 @@ Remove [DVC-files](/doc/user-guide/dvc-file-format) from your repository.
 It removes DVC-files, and the entire `.dvc/` meta directory from the workspace.
 Note that the DVC cache will normally be removed as well, unless it's set to an
 external location with `dvc cache dir`. (By default a local cache is located in
-the `.dvc/cache` directory.)
+the `.dvc/cache` directory.) If you were using symlinks for linking data from
```

Comment:
put a link here `[symlinks for linking data]/doc/user-guide/large-dataset-optimization`